nipype/interfaces/ants/registration.py

```diff
@@ -1596,3 +1596,59 @@ def _list_outputs(self):
             outputs['forward_warp_field'] = out_base + '1Warp.nii.gz'
             outputs['inverse_warp_field'] = out_base + '1InverseWarp.nii.gz'
         return outputs
+
+class CompositeTransformUtilInputSpec(ANTSCommandInputSpec):
+    process = traits.Enum('assemble', 'disassemble', argstr='--%s',
+        position=1, usedefault=True,
+        desc='What to do with the transform inputs (assemble or disassemble)',
+        )
+    in_file = InputMultiPath(File(exists=True), mandatory=True, argstr='%s...',
+        position=2, desc='Input transform file(s)')
+    output_prefix = Str("transform", usedefault=True, argstr='%s',
+        position=3, desc="A prefix that is prepended to all output files")
+
+class CompositeTransformUtilOutputSpec(TraitedSpec):
+    affine_transform = File(exists=True, desc="Affine transform component",
+            mandatory=True, position=2)
+    displacement_field = File(desc="Displacement field component")
+
+class CompositeTransformUtil(ANTSCommand):
+    """
+    ANTs utility which can combine or break apart transform files into their individual
+    constituent components.
+
+    Examples
+    --------
+
+    >>> from nipype.interfaces.ants import CompositeTransformUtil
+    >>> tran = CompositeTransformUtil()
+    >>> tran.inputs.process = 'disassemble'
+    >>> tran.inputs.in_file = 'output_Composite.h5'
+    >>> tran.cmdline
+    'CompositeTransformUtil --disassemble output_Composite.h5 transform'
+    >>> tran.run()  # doctest: +SKIP
+    """
```

Comment:
Maybe add an example with `'assemble'`, as well, since that changes the use of `process`. That will both make clearer how the interface works and make sure we exercise that bit of code when running tests.